tonic_validate/metrics/augmentation_accuracy_metric.py

```diff
@@ -12,6 +12,13 @@
 class AugmentationAccuracyMetric(Metric):
     name: str = "augmentation_accuracy"
 
+    def __init__(self):
+        """
+        Metric that checks whether all the context is in the LLM answer.
+        Returns a float between 0 and 1, where 1 is it contains all the context and 0 is it contains none of the context.
```

Comment:
Returns a float between 0 and 1. 1 indicates that the answer contains all of the context. 0 indicates that it contains none of the context.